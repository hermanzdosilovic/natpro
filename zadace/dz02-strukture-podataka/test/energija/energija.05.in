1000
ADD 787 835
ADD 629 873
ADD 259 1964
ADD 467 350
ADD 977 1702
ADD 296 2684
ADD 350 2164
ADD 538 1548
ADD 196 2212
ADD 317 3503
ADD 735 2834
ADD 490 3071
ADD 545 1875
ADD 878 2407
ADD 290 1761
ADD 808 2839
ADD 640 2279
ADD 100 2049
ADD 936 2783
ADD 167 970
ADD 514 3866
ADD 14 1372
ADD 364 15
ADD 472 3026
ADD 897 1325
ADD 429 1045
ADD 593 2243
ADD 320 1900
ADD 841 781
ADD 952 3399
ADD 979 2781
ADD 120 2015
ADD 614 2930
ADD 334 2983
ADD 176 1482
ADD 230 637
ADD 500 2368
ADD 209 2260
ADD 768 53
ADD 162 1829
ADD 801 1951
ADD 154 3945
ADD 971 507
ADD 588 3293
ADD 862 3060
ADD 851 1327
ADD 738 1397
ADD 139 2064
ADD 872 765
ADD 884 799
ADD 978 3074
ADD 982 1014
ADD 440 3702
ADD 379 3683
ADD 366 3528
ADD 479 3288
ADD 868 1140
ADD 475 2048
ADD 776 915
ADD 905 653
ADD 782 2696
ADD 561 1547
ADD 539 3728
ADD 158 369
ADD 718 1720
ADD 470 3942
ADD 448 2220
ADD 80 194
ADD 940 2086
ADD 23 759
ADD 728 1872
ADD 16 2449
ADD 727 1275
ADD 464 2760
ADD 27 3779
ADD 819 2048
ADD 396 188
ADD 46 3286
ADD 21 3728
ADD 215 279
ADD 744 3845
ADD 119 1293
ADD 278 533
ADD 980 3543
ADD 925 1823
ADD 424 3034
ADD 488 2279
ADD 805 1273
ADD 164 3946
ADD 101 2949
ADD 460 2231
ADD 959 3590
ADD 781 2082
ADD 530 1685
ADD 190 1296
ADD 521 2813
ADD 613 188
ADD 363 2385
ADD 631 1107
ADD 673 3224
ADD 690 701
ADD 30 737
ADD 934 1256
ADD 93 2511
ADD 135 1019
ADD 503 1677
ADD 723 1052
ADD 35 3712
ADD 891 3182
ADD 854 3962
ADD 584 2675
ADD 549 1700
ADD 904 95
ADD 599 1264
ADD 733 2289
ADD 845 2349
ADD 271 3288
ADD 121 1752
ADD 945 3995
ADD 97 1857
ADD 775 2988
ADD 385 3981
ADD 997 3522
ADD 885 1632
ADD 594 3892
ADD 864 1185
ADD 831 2199
ADD 86 8
ADD 42 1922
ADD 238 1083
ADD 832 901
ADD 457 1861
ADD 921 918
ADD 779 1876
ADD 362 1346
ADD 146 423
ADD 166 2304
ADD 707 2429
ADD 912 3152
ADD 687 145
ADD 696 3550
ADD 171 3128
ADD 693 2398
ADD 417 1586
ADD 995 2497
ADD 229 1961
ADD 796 711
ADD 699 3055
ADD 251 3741
ADD 231 81
ADD 495 2991
ADD 821 119
ADD 791 3719
ADD 578 2009
ADD 601 465
ADD 658 262
ADD 383 2134
ADD 242 729
ADD 600 1996
ADD 19 3150
ADD 721 13
ADD 85 2814
ADD 709 2678
ADD 681 1010
ADD 833 1931
ADD 75 3378
ADD 155 3606
ADD 968 1790
ADD 815 1267
ADD 361 3331
ADD 732 1410
ADD 636 3043
ADD 439 3948
ADD 159 2378
ADD 947 3296
ADD 452 3600
DEL 139
QUERY 19222
DEL 815
QUERY 23817
DEL 718
QUERY 22989
DEL 709
QUERY 33346
DEL 768
QUERY 4449
DEL 530
QUERY 41198
DEL 429
QUERY 43639
DEL 259
QUERY 29588
DEL 464
QUERY 23677
DEL 891
QUERY 21617
DEL 945
QUERY 41053
DEL 27
QUERY 13632
DEL 696
QUERY 10194
DEL 921
QUERY 46741
DEL 539
QUERY 4760
DEL 600
QUERY 22203
DEL 594
QUERY 32773
DEL 721
QUERY 9423
DEL 779
QUERY 22678
DEL 385
QUERY 36216
DEL 30
QUERY 47781
DEL 42
QUERY 44219
DEL 699
QUERY 602
DEL 379
QUERY 49617
DEL 673
QUERY 7154
DEL 120
QUERY 1884
DEL 155
QUERY 11201
DEL 599
QUERY 31233
DEL 636
QUERY 20397
DEL 271
QUERY 15553
DEL 162
QUERY 7542
DEL 681
QUERY 34530
DEL 805
QUERY 25379
DEL 119
QUERY 33620
DEL 631
QUERY 11547
DEL 796
QUERY 2845
DEL 424
QUERY 40134
DEL 166
QUERY 8760
DEL 242
QUERY 46868
DEL 775
QUERY 20880
DEL 470
QUERY 49259
DEL 728
QUERY 5718
DEL 979
QUERY 49357
DEL 521
QUERY 47256
DEL 912
QUERY 37497
DEL 14
QUERY 20114
DEL 215
QUERY 28533
DEL 251
QUERY 46400
DEL 561
QUERY 36159
DEL 707
QUERY 20269
DEL 488
QUERY 35366
DEL 231
QUERY 3657
DEL 997
QUERY 4923
DEL 831
QUERY 3622
DEL 167
QUERY 4775
DEL 500
QUERY 39361
DEL 640
QUERY 6930
DEL 732
QUERY 25834
DEL 472
QUERY 38631
DEL 925
QUERY 45150
DEL 735
QUERY 7489
DEL 101
QUERY 28037
DEL 490
QUERY 37197
DEL 452
QUERY 7422
DEL 819
QUERY 18141
DEL 549
QUERY 3486
DEL 723
QUERY 6000
DEL 159
QUERY 21289
DEL 290
QUERY 46721
DEL 46
QUERY 42652
DEL 538
QUERY 36733
DEL 872
QUERY 34646
DEL 362
QUERY 30980
DEL 782
QUERY 35330
DEL 229
QUERY 20403
DEL 93
QUERY 18432
DEL 971
QUERY 46953
DEL 982
QUERY 38050
DEL 841
QUERY 3080
DEL 86
QUERY 36490
DEL 864
QUERY 40437
DEL 578
QUERY 3142
DEL 588
QUERY 9751
DEL 164
QUERY 14307
DEL 613
QUERY 31213
DEL 801
QUERY 8405
DEL 361
QUERY 40690
DEL 862
QUERY 18292
DEL 366
QUERY 12899
ADD 680 2477
ADD 76 2130
ADD 26 2875
QUERY 42236
QUERY 1382
ADD 46 3142
QUERY 14237
ADD 677 1026
ADD 505 1366
QUERY 41557
QUERY 18536
QUERY 21957
QUERY 33607
ADD 831 1158
QUERY 25741
ADD 5 2720
QUERY 7015
ADD 181 216
ADD 538 2893
QUERY 45891
ADD 932 1841
QUERY 17225
QUERY 26074
QUERY 36963
ADD 128 405
QUERY 12316
QUERY 26237
ADD 645 2184
ADD 73 605
QUERY 660
ADD 25 1186
QUERY 13596
QUERY 43548
QUERY 18052
QUERY 45867
ADD 131 2887
ADD 707 2310
QUERY 16462
QUERY 10588
ADD 745 1997
ADD 499 195
QUERY 23051
QUERY 6463
ADD 311 2013
ADD 536 1078
QUERY 39177
QUERY 266
QUERY 21703
QUERY 49005
ADD 172 662
ADD 199 3354
DEL 467
QUERY 25162
DEL 23
ADD 671 2478
ADD 900 1663
QUERY 23572
ADD 697 1954
QUERY 48217
ADD 434 133
ADD 886 2135
QUERY 35194
DEL 905
QUERY 38523
DEL 363
QUERY 48939
QUERY 49695
QUERY 27367
ADD 225 1496
QUERY 12812
QUERY 47754
QUERY 35742
ADD 963 874
DEL 73
QUERY 15717
QUERY 21102
ADD 623 2050
ADD 692 731
ADD 51 2287
ADD 698 3662
ADD 375 2646
QUERY 46341
QUERY 22582
ADD 510 905
QUERY 46264
QUERY 32982
QUERY 34126
QUERY 49864
QUERY 5037
QUERY 13748
QUERY 144
QUERY 17483
QUERY 33634
DEL 629
ADD 466 3751
QUERY 5890
ADD 660 1337
ADD 221 2380
QUERY 45338
QUERY 49443
QUERY 19064
ADD 1 3484
QUERY 12604
DEL 932
QUERY 15416
QUERY 20002
QUERY 23980
QUERY 31023
ADD 268 2049
QUERY 49262
ADD 473 3480
QUERY 10174
QUERY 23227
QUERY 18747
DEL 968
ADD 917 2319
QUERY 44591
QUERY 21345
ADD 750 2394
DEL 19
DEL 781
ADD 537 276
QUERY 14463
QUERY 43105
ADD 923 1765
QUERY 44036
ADD 575 2580
DEL 154
DEL 936
ADD 989 1028
ADD 706 1235
ADD 152 2309
DEL 158
DEL 176
QUERY 3411
DEL 26
DEL 776
QUERY 34302
QUERY 13531
QUERY 32945
QUERY 12679
ADD 986 1149
QUERY 21715
DEL 693
ADD 789 618
QUERY 17163
QUERY 46175
QUERY 43150
DEL 660
DEL 85
QUERY 43363
ADD 490 304
QUERY 17809
QUERY 13305
QUERY 1960
ADD 419 831
ADD 617 3549
ADD 149 1460
ADD 560 889
DEL 5
QUERY 35400
QUERY 1144
ADD 397 1356
QUERY 23648
QUERY 48159
ADD 563 2810
QUERY 18294
QUERY 42941
DEL 100
QUERY 27937
ADD 348 3803
QUERY 33597
QUERY 1092
QUERY 40139
ADD 902 2716
QUERY 45615
ADD 987 1313
QUERY 40634
QUERY 4648
ADD 913 2770
QUERY 4324
ADD 215 1227
ADD 762 1889
QUERY 9523
QUERY 19540
ADD 751 2470
ADD 590 2376
QUERY 42112
QUERY 26419
QUERY 11873
QUERY 46093
QUERY 19910
QUERY 23067
ADD 429 1814
QUERY 31749
ADD 459 1185
ADD 574 3874
QUERY 17643
ADD 101 1999
QUERY 28376
QUERY 40088
QUERY 30874
QUERY 41135
QUERY 7941
QUERY 4589
ADD 351 1379
QUERY 23320
QUERY 31223
QUERY 3319
ADD 436 1752
ADD 807 815
QUERY 17431
QUERY 48077
QUERY 24671
QUERY 1322
QUERY 28287
QUERY 5567
ADD 288 2313
QUERY 1602
QUERY 15231
ADD 731 3634
QUERY 10907
QUERY 3096
ADD 767 987
QUERY 2020
ADD 226 1226
QUERY 21967
QUERY 15380
QUERY 24866
DEL 934
QUERY 1212
DEL 978
QUERY 11937
QUERY 36024
QUERY 26638
ADD 588 2387
QUERY 12667
QUERY 29679
QUERY 48980
QUERY 25452
DEL 320
QUERY 41205
ADD 711 1922
ADD 998 783
DEL 473
QUERY 41187
ADD 559 915
QUERY 24477
QUERY 33675
ADD 249 2684
QUERY 6145
ADD 675 274
QUERY 14616
QUERY 3301
QUERY 10588
DEL 351
DEL 807
QUERY 20026
QUERY 24515
QUERY 39257
QUERY 26931
ADD 392 3411
ADD 504 3909
ADD 345 2679
ADD 515 3402
DEL 690
QUERY 22646
ADD 668 2272
ADD 652 3258
ADD 206 3080
QUERY 863
DEL 536
QUERY 19674
QUERY 35573
QUERY 43301
QUERY 20709
QUERY 10156
ADD 763 2791
QUERY 43132
QUERY 20775
ADD 602 1168
QUERY 45682
ADD 20 937
ADD 365 257
DEL 617
ADD 739 1682
QUERY 22688
DEL 658
QUERY 24776
QUERY 1228
QUERY 36331
ADD 315 3935
QUERY 12051
QUERY 32368
QUERY 4314
DEL 230
QUERY 13044
DEL 588
QUERY 21706
QUERY 39926
ADD 265 781
DEL 884
QUERY 1694
QUERY 11860
ADD 588 1138
QUERY 1982
QUERY 38992
QUERY 17455
QUERY 24334
ADD 876 2422
QUERY 36350
QUERY 12623
QUERY 41684
DEL 559
QUERY 46954
ADD 642 1022
QUERY 46293
QUERY 45075
QUERY 14907
QUERY 43307
ADD 631 3642
ADD 286 3113
ADD 321 2828
QUERY 8510
QUERY 25936
QUERY 17932
QUERY 9947
QUERY 16706
ADD 412 701
QUERY 41036
QUERY 20904
QUERY 29249
ADD 387 3343
QUERY 10143
QUERY 22745
ADD 647 123
QUERY 5180
QUERY 11651
ADD 812 1734
QUERY 49664
QUERY 15985
QUERY 5837
QUERY 10954
ADD 982 2600
ADD 352 1057
QUERY 20475
EXIT
