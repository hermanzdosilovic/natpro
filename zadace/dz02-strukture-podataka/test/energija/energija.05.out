16
16
16
16
14
19
19
16
16
16
19
14
14
19
14
16
16
14
16
19
19
19
14
21
14
14
14
16
16
14
14
19
16
16
14
14
19
14
19
16
19
14
19
19
19
19
21
21
21
19
21
16
16
16
16
21
16
19
21
21
16
21
21
16
19
16
16
19
21
21
21
21
21
21
19
19
21
21
16
21
21
16
16
19
21
16
21
19
16
21
16
19
21
19
19
21
19
16
21
19
19
21
16
19
5
19
21
19
21
19
16
19
16
21
5
19
21
19
19
21
21
21
21
21
21
16
21
21
19
19
21
19
21
21
21
21
16
19
5
19
21
16
21
21
19
5
5
19
19
21
21
5
19
16
21
19
5
21
21
1
21
5
21
5
16
16
21
21
21
16
5
1
21
1
16
21
16
21
21
21
1
21
21
21
1
1
1
16
21
16
1
21
16
16
21
16
21
21
21
21
1
1
16
21
1
16
21
16
1
21
1
1
1
1
1
1
16
1
16
1
1
21
16
1
21
21
16
21
21
16
21
1
1
1
1
16
16
21
21
16
1
16
21
21
16
1
21
16
21
16
16
1
16
1
16
1
1
16
16
1
1
1
16
16
16
16
1
16
16
16
16
1
16
1
16
16
1
16
16
16
16
1
16
1
1
16
1
1
1
16
